<launch>
    <node pkg="arx_r5pro_controller" type="R5ProController" name="vr_slave_l" output="screen">
        <param name="arm_control_type" value="normal_v1"/>
        <param name="arm_can_id" value="can1"/>
        <param name="arm_end_type" value="0"/>

        <param name="sub_topic_name" value="/ARX_VR_L"/>
        <param name="pub_topic_ee_name_v1" value="/follow1_pos_back"/>
        <param name="pub_topic_joint_name_v1" value="/joint_information"/>
    </node>

    <node pkg="arx_r5pro_controller" type="R5ProController" name="vr_slave_r" output="screen">
        <param name="arm_control_type" value="normal_v1"/>
        <param name="arm_can_id" value="can3"/>
        <param name="arm_end_type" value="0"/>

        <param name="sub_topic_name" value="/ARX_VR_R"/>
        <param name="pub_topic_ee_name_v1" value="/follow2_pos_back"/>
        <param name="pub_topic_joint_name_v1" value="/joint_information2"/>
    </node>
</launch>